exe/IHP/IDE/DevServer.hs

```diff
@@ -175,9 +179,10 @@ stop AppState { .. } = do
     stopFileWatcher fileWatcherState
     stopToolServer toolServerState
 
-startFilewatcher :: (?context :: Context) => IO ()
-startFilewatcher = do
-        thread <- async $ FS.withManager $ \manager -> do
+startFileWatcher :: (?context :: Context) => IO ()
+startFileWatcher = do
+        let watchConfig = ?context |> get #fileWatcherConfig
```

Comment:
Didn't know that the file watcher library supports this out of the box :) really nice.

Any reasons you didn't move the

```haskell
    let fileWatcherDebounceTime = Clock.secondsToNominalDiffTime 0.1 -- 100ms
    let fileWatcherConfig = FS.defaultConfig { FS.confDebounce = FS.Debounce fileWatcherDebounceTime }
```

directly into the `startFileWatcher`?

This way we can avoid having the `, fileWatcherConfig :: WatchConfig` in the app state, which makes this a bit simpler.